tsconfig.json

```diff
@@ -0,0 +1,37 @@
+
+{
+	"compilerOptions": {
+        // Forked from https://github.com/sindresorhus/tsconfig/blob/640e4bfe1172e27d59bf4d73a2c5df517cc01e1d/tsconfig.json
+
```

Comment:
Please check the configs in https://github.com/apify/apify-storage-local-js and https://github.com/apify/browser-pool

Reply:
So many config files 🥶

Reply:
We could make this a module, like the eslint config! Just need to do it 👍

Reply:
Update: it's been done! @apify/tsconfig is available, check out browser-pool for how to use it 😋. If you want, I can push it for you